packages/bitcore-wallet-service/test/integration/server.js

```diff
@@ -3172,7 +3172,7 @@ describe('Wallet service', function() {
             var txOpts = {
               outputs: [{
                 toAddress: addressStr,
-                amount: 0,
```

Comment:
why did this change to -1?

Reply:
ETH txs can have amount of 0

Reply:
This would also allow BTC txs with 0 amount. Need to discuss more

Reply:
since this is for testing, and we want to keep the amount=0 test for BTC, maybe do:
         `   amount: (coin == 'eth') ? -1 :  0 `